share/module_check_a_mundo.F

```diff
@@ -2298,6 +2298,26 @@ END FUNCTION bep_bem_nbui_max
         count_fatal_error = count_fatal_error + 1
       END IF
 
+!-----------------------------------------------------------------------
+!  Consistency checks for vertical refinement:
+!  rebalance must be set to 1 
+!-----------------------------------------------------------------------
+      oops = 0 
+      DO i = 2, model_config_rec % max_dom
+        IF ( .NOT. model_config_rec % grid_allowed(i) ) CYCLE
+        IF (model_config_rec%vert_refine_method(i) .NE. 0) THEN
+          IF ( model_config_rec%rebalance .NE. 1 ) THEN
+            oops = oops + 1
+          END IF
+        END IF
+      END DO
+
+      IF ( oops .GT. 0 ) THEN
+        wrf_err_message = '--- ERROR: vert_refine_method=2 only works with rebalance=1 '
+        CALL wrf_debug ( 0, TRIM( wrf_err_message ) )
+        count_fatal_error = count_fatal_error + 1
+      END IF
```

Comment:
@weiwangncar @dudhia @rsarthur
Wei and Jimy,
What does everyone think about setting rebalance to 1?
Basically, IF vert_refine_method = 2, THEN set rebalance to 1.

Also, do we need to worry about vert_refine_method != to 0, OR only vert_refine_method = 1?

Reply:
> Also, do we need to worry about vert_refine_method != to 0, OR only vert_refine_method = 1?

Wrong value, I meant "2".
Also, do we need to worry about vert_refine_method != to 0, OR only vert_refine_method = 2?

Reply:
Adding @kalundquist
I think we want vert_refine_method != 0, because both 1 and 2 should require rebalancing. We could change the error message to include both options. If so, we might want to change other similar error messages in check_a_mundo, which I copied this one from.